### PR TITLE
docs: miscellaneous updates (edge)

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR:

- Adds an extra step for overlay enablement
- Adds extra context for registry parameter
- Fixes prerequisites

## Review Changes

💻 [Add Preview URL]()

🎫 [Jira Ticket]()

### CODE REVIEW

File: docs/docs-content/clusters/edge/networking/vxlan-overlay.md [hunk 1]
```diff
@@ -103,19 +112,19 @@ You will not be able to change the network overlay configurations after the clus
...
+9. Review the rest of your cluster profile values and make changes as needed. Click on **Next**.
 
-9. In the **Cluster Config** stage, toggle on **Enable Overlay Network**. This will prompt you to provide additional configuration for your virtual overlay network. 
+10. In the **Cluster Config** stage, toggle on **Enable Overlay Network**. This will prompt you to provide additional configuration for your virtual overlay network. 
 
-10. In the **Overlay CIDR Range** field, provide a private IP range for your cluster to use. Ensure that this range is not used by others in the same network environment. When you toggle on **Enable Overlay Network**, Palette provides with a default commonly unused range. We suggest you keep the default range unless you have a specific IP range you want to use. 
+11. In the **Overlay CIDR Range** field, provide a private IP range for your cluster to use. Ensure that this range is not used by others in the same network environment. When you toggle on **Enable Overlay Network**, Palette provides with a default commonly unused range. We suggest you keep the default range unless you have a specific IP range you want to use.
```
Comment (ritawatson) [inline docs/docs-content/clusters/edge/networking/vxlan-overlay.md:119]:
```suggestion
11. In the **Overlay CIDR Range** field, provide a private IP range for your cluster to use. Ensure this range is not used by others in the same network environment. When you toggle on **Enable Overlay Network**, Palette provides a default range that is commonly unused. We suggest you keep the default range unless you have a specific IP range you want to use. 
```

File: docs/docs-content/clusters/edge/networking/vxlan-overlay.md [hunk 2]
```diff
@@ -125,7 +134,7 @@ You will not be able to change the network overlay configurations after the clus
 
 3. Select the host cluster you created to view its details page.
 
-4. Select the **Nodes** tab, in the **Overlay IP Address** column, each host has an overlay IP address within the CIDR range you provided during cluster configuration. 
+4. Select the **Nodes** tab, in the **Private Ips** column, each host has an overlay IP address within the CIDR range you provided during cluster configuration.
```
Comment (ritawatson) [inline docs/docs-content/clusters/edge/networking/vxlan-overlay.md:137]: This is a run-on sentence. Is the following intended?

Select the **Nodes** tab. For each host, the **Private Ips** column lists an overlay IP address within the CIDR range you provided during cluster configuration.

File: docs/docs-content/integrations/byoos.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ The BYOS Edge OS pack supports the following parameters.
 | `pack.content.images` | Specifies a list of OS images to use with the pack. | list |
 | `pack.content.images.image` | An OS image to use with the pack. | string|
 | `system.uri` | The system URI specifies the location of BYOOS image. | string|  
-| `providerCredentials.registry` | Specifies the private registry for the cluster to pull images from | string |  
+| `providerCredentials.registry` | Specifies the private registry for the cluster to pull images from. If you are using a Harbor registry, provide the Harbor domain as well as the name of the project. For example, `harbor.spectrocloud.com/default`. If you are using a Docker registry, provide the domain of the registry. For example, `registry-1.docker.io`. | string |
```
Comment (ritawatson) [inline docs/docs-content/integrations/byoos.md:55]:
```suggestion
| `providerCredentials.registry` | Specifies the private registry for the cluster to pull images from. If you are using a Harbor registry, provide the Harbor domain and the project's name. For example, `harbor.spectrocloud.com/default.` If you are using a Docker registry, provide the domain of the registry. For example, `registry-1.docker.io`. | string |  
```

File: docs/docs-content/registries-and-packs/spectro-cli-reference.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ The Spectro CLI tool is a command-line interface to interact with Palette regist
 
 :::info
 
-We recommend using an OCI registry for storing and maintaining your packs. Refer to the [OCI registry](oci-registry.md) section for more information.
+We recommend using an OCI registry for storing and maintaining your packs. Refer to the [OCI registry](registries/oci-registry/oci-registry.md) section for more information.
```
Comment (ritawatson) [inline docs/docs-content/registries-and-packs/spectro-cli-reference.md:16]:
```suggestion
We recommend using an OCI registry to store and maintain your packs. Refer to the [OCI registry](registries/oci-registry/oci-registry.md) section for more information.
```

**Review by ritawatson — APPROVED** [review]
Some minor suggestions to address. Otherwise, LGTM.

File: docs/docs-content/clusters/edge/networking/vxlan-overlay.md (hunk 2 above)
Comment (lennessyy) [inline docs/docs-content/clusters/edge/networking/vxlan-overlay.md:137]:
```suggestion
4. Select the **Nodes** tab. For each host, the **Private Ips** column lists an overlay IP address within the CIDR range you provided during cluster configuration.
```

**Pull request merged**